Packs/CommonTypes/IncidentFields/incidentfield-State.json

```diff
@@ -0,0 +1,44 @@
+{
+	"associatedToAll": true,
+	"associatedTypes": null,
+	"breachScript": "",
+	"caseInsensitive": true,
+	"cliName": "state",
+	"closeForm": false,
+	"columns": null,
+	"content": true,
+	"defaultRows": null,
+	"description": "State",
+	"editForm": true,
+	"fieldCalcScript": "",
+	"fromServerVersion": "5.0.0",
```

Comment:
did we run format on this file? we can remove a lot of fields from here such as `fromServerVersion`